server/templatize.js

```diff
@@ -3,7 +3,7 @@
 let Nunjucks = require("nunjucks");
 
 module.exports = function templatize(server, paths) {
-  let engine = new Nunjucks.Environment(paths.map(path => new Nunjucks.FileSystemLoader(path)), { autoescape: true });
+  let engine = new Nunjucks.Environment(paths.map(path => new Nunjucks.FileSystemLoader(path, { noCache: true, watch: true } )), { autoescape: true });
```

Comment:
Note that `{autoescape: true}` is already the nunjucks options object, so noCache and watch [can go straight in that](https://mozilla.github.io/nunjucks/api.html#configure). Also, this line is getting a bit long, it would be good to change it to:
```
let nunjucksOptions = {
  noCache: true,
  watch: true,
  autoescape: true
};

let engine = new Nunjucks.Environment(paths.map(path => {
  return new Nunjucks.FileSystemLoader(path, nunjucksOptions);
}
```